### DISCUSSION TITLE
range slider want to use in ARABIC

### DISCUSSION THREAD

**Discussion by ramanvermag** [andreruffert/rangeslider.js#367] [Q&A] [answered]
how I can use this in ARABIC language like Right to left language

**Comment by andreruffert** [comment, marked as answer]
rangeslider.js does not support directionality. 

You could use a more modern rewrite like [&lt;range-slider&gt;](https://github.com/andreruffert/range-slider-element) element which supports this functionality out of the box by setting the [dir](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir) attribute. 

👉 https://github.com/andreruffert/range-slider-element#attributes
👀 https://codepen.io/andreruffert/pen/MWpdqPZ

I hope this helps! ✌️
